<?xml version="1.0"?>

<launch>
<param name="use_sim_time" value="false"/>
<arg name="drone_name" default="iris"/>
  <group ns="$(arg drone_name)">
    <node name="vicon_compute_relative_state_moving_platform_drone_node" pkg="training_q_learning" type="vicon_compute_relative_state_moving_platform_drone.py" output="screen" >
        <param name = "vicon_relative_state_moving_platform_drone_hz" value = "100" />
        <param name = "drone_name" value = "$(arg drone_name)" />
    </node> 
  </group>  
</launch>
